jgiven-core/src/test/java/com/tngtech/jgiven/report/ReportConfigArgumentTest.java

```diff
@@ -0,0 +1,29 @@
+package com.tngtech.jgiven.report;
+
+import com.tngtech.jgiven.report.config.ConfigOption;
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReportConfigArgumentTest {
```

Comment:
This is the former `ReportGeneratorArgumentTest` (next file; deleted). Like the test further down after that, it does not really require the asciidoc reporter, so I replaced it with anonymous classes. It might be worthwhile extracting those or moving/renaming the tests to signify that they are really testing the `AbstractReportConfig` and `AbstractReportGenerator`, respectively.